gfw_pixetl/utils/gdal.py

```diff
@@ -133,14 +133,21 @@ def run_gdal_subcommand(
         o = str(o_byte)
         e = str(e_byte)
 
-    # Sometimes GDAL returns exit code 0 even though there was actually an error
-    # It's hard to tell what is and what isn't though, so log as a warning
+    # Sometimes GDAL returns exit code 0 even though there was actually an error.
+    # Raise a hard error if it is an SSL SYSCALL error, since that possibly means a
+    # database connection was dropped (and so a tile rasterization might be
+    # incomplete).
+    if p.returncode == 0 and "ERROR 1: SSL SYSCALL error" in e:
```

Comment:
minor, but I think it would look a little neater to just combine this if statement and the one below into a nested if statement under `if p.returncode == 0`, and then check those two subconditions inside there?

Reply:
Sure, sounds good.  Made that change.  Thanks!